src/binutil/setup.rs

```diff
@@ -31,10 +32,27 @@ pub fn initial_logger(config: &TiKvConfig) {
     let log_rotation_timespan =
         chrono::Duration::from_std(config.log_rotation_timespan.clone().into())
             .expect("config.log_rotation_timespan is an invalid duration.");
+
+    // Collects following targets.
+    const ENABLED_TARGETS: &[&str] = &[
```

Comment:
oh, seem now we have not filtered any message, right? @BusyJay

Reply:
Yes. And it's expected not to filter any messages. Here is just for debug purpose, will remove later.